apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -389,11 +676,9 @@ static void itc_wifi_manager_reconnect_p(void)
 	TC_ASSERT_EQ_CLEANUP("wifi_manager_connect_ap", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
 	WIFITEST_WAIT;
 
-	sleep(SEC_5);
 	ret = wifi_manager_connect_ap(&config);
 	TC_ASSERT_EQ_CLEANUP("wifi_manager_connect_ap", ret, WIFI_MANAGER_SUCCESS, wifi_manager_disconnect_ap(); wifi_manager_deinit());
 
-	sleep(SEC_5);
 	ret = wifi_manager_disconnect_ap();
 	TC_ASSERT_EQ_CLEANUP("wifi_manager_disconnect_ap", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
 
```

Comment:
Please add WIFITEST_WAIT here, to prevent a state error.

Reply:
Then, you can call itc_wifi_manager_reconnect_p() at the end of the main function.